lang/rust/patches/0002-rustc-bootstrap-cache.patch

```diff
@@ -23,10 +23,10 @@
          let cache_dir = cache_dst.join(key);
          if !cache_dir.exists() {
              t!(fs::create_dir_all(&cache_dir));
-@@ -627,7 +630,10 @@ download-rustc = false
+@@ -647,7 +650,10 @@ download-rustc = false
          let llvm_assertions = self.llvm_assertions;
  
-         let cache_prefix = format!("llvm-{}-{}", llvm_sha, llvm_assertions);
+         let cache_prefix = format!("llvm-{llvm_sha}-{llvm_assertions}");
 -        let cache_dst = self.out.join("cache");
 +        let cache_dst = match env::var_os("OPENWRT_RUSTC_BOOTSTRAP_CACHE") {
```

Comment:
Probably we could float it towards upstream using a less os-specific name

Reply:
I'll put there whatever you want- that is just refreshed patch that appeared in previous package release update (1.72.0-3) without me doing any actual changes.